tools/deploy.sh

```diff
@@ -44,6 +44,10 @@ IRONIC_DEPLOY_FILES="${SCRIPTDIR}/ironic-deployment/basic-auth/default/auth.yaml
 	${SCRIPTDIR}/ironic-deployment/tls/keepalived/kustomization.yaml \
 	${SCRIPTDIR}/ironic-deployment/tls/keepalived/tls.yaml"
 
+if [ ! -f ${KUSTOMIZE} ]; then
```

Comment:
There's no need to make this conditional, checking whether the target is up to date is kinda what make does.